doc/sphinxman/source/glossary_psivariables.rst

```diff
@@ -1487,7 +1487,8 @@ PSI Variables by Alpha
    irrep are as in :ref:`sec:psivarnotes`.
 
 
-.. psivar:: TD-fctl ROOT 0 -> ROOT m ELECTRIC TRANSITION DIPOLE MOMENT (LEN)
+.. psivar:: ADC ROOT 0 -> ROOT m ELECTRIC TRANSITION DIPOLE MOMENT (LEN)
```

Comment:
Don't forget to document the other two access patterns for all of these.

Reply:
Done.